testdata/stringLiteral/cm.yaml

```diff
@@ -3,4 +3,25 @@ kind: ConfigMap
 metadata:
   name: myapp
 data:
-  key: "{\"uid\": \"${datasource}\"}"
\ No newline at end of file
+  key: "{\"uid\": \"$(datasource)\"}"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: myapp-var
+data:
+  key: "{\"uid\": \"${datasource}\"}"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: myapp-no-end-bracket
+data:
+  key: "{\"uid\": \"${datasource\"}"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: myapp-no-brackets
+data:
+  key: "{\"uid\": \"${datasource\""
```

Comment:
Did you intend this example to be "no brackets", like `"{\"uid\": \"$datasource\"}"`?

Reply:
Good catch. Updated!